Makefile

```diff
@@ -134,6 +134,36 @@ else
 	go test -v -timeout 3600s $(TEST_ARGS) ./test/e2e/...
 endif
 
+kind-e2e-test: build kustomize docker-build create-kind-multicluster kind-load-image-multi e2e-test
+
+single-up: build kustomize docker-build create-kind-cluster kind-load-image cert-manager
```

Comment:
Ignore my other comment on this in the multi-cluster area since you explained here @jsanda !

We can discuss here... If we don't do the delete here, would it redeploy things to the existing kind clusters? If that's the case, then that makes total sense.  Although I am lazy and don't want to type `kind delete cluster --name` 😃

Reply:
I'm also lazy and want everything to be rebuilt without having to type that damned `kind delete` command.
Shouldn't we then make the operator redeployment another target then? Something like `make upgrade-operator` ?

Reply:
Agreed. I think `reload-operator` is clearer. I would still prefer deletion to be explicit and I still prefer `single-up` and `multi-up` to be additive only.

Reply:
Works for me